Packs/Auditd/ParsingRules/Auditd/Auditd.yml

```diff
@@ -0,0 +1,6 @@
+name: UnixAuditd Parsing Rule
+id: UnixAuditd Parsing Rule
```

Comment:
not sure that spaces are allowed in the ID

Reply:
yes i checked it with other Parsing rules packs